index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1>Hello World! we are software developers!</h1>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img src="Lighthouse.jpg" alt="image" id="image" />
+        <h2 id="img-caption">this is testing image </h2>
+        <div id="tribute-info">
+          
+
+          <h1>Date: Wendsday</h1> <br>
+          <h1>Date: Wendsday</h1><br>
+          <h1>Date: Wendsday</h1><br>
+          <h1>Date: Wendsday</h1><br>
+          <h1>Date: Wendsday</h1><br>
+          <h1>Date: Wendsday</h1><br>
+          <h1>Date: Wendsday</h1><br>
+          
+
+
+        this page is collaborating project in microverse startin html and css
+        </div>
```

Comment:
plese, rearrange this section ul and li tags to follow the conventional best practices. you can read about this at HTML & CSS best practices.
also, ger read of the extra white spaces. they could make your code look bad.